### ISSUE TITLE
timing out? chrome developer render process gone 

### ISSUE THREAD

**Issue by tmblue** [Lyfhael/DeleteTweets#74]
Any way to adjust this? It keeps popping up with the error and forcing me to restart, I am trying from a dump (it's huge 42K), so will run for 5-10 minutes and bail 

Debugging connection was closed..

I have the window open in it's own window I can see it's incrementally going through the tweets, it's not sleeping or losing a connection that I can tell.

Maybe there is a timeout i can set? But so far everything seems to be working :)

**Comment by justex07** [comment]
I am having this same issue when it hits around 16k tweets deleted. The error I receive is "Debugging connection was closed. Reason: Render process gone. Reconnect when ready by reopening DevTools."

It doesn't appear that restarting it picks up where it left off which is a bummer.

![Google Chrome-DevTools - x comhome-2024-11-15 at 13 51 48@2x](https://github.com/user-attachments/assets/6275071b-4577-48ba-ab89-85daa11a7f59)

**Comment by n3wjack** [comment]
As it says in the message, try closing & reopening the DevTools windows (F12 normally).
You can retrieve your previous script by pressing the arrow up key in the console window & hit Enter.

If that doesn't work, restarting the browser, and setting up the script again should.

**Comment by tmblue** [comment]
No, why do you think that is the solution. Sure we can start over, we have the script, but it's going to start on the same script and time out again, doing the same thing over and over again is a sign of something :)

**Comment by justex07** [comment]
I ran it again today at 81k tweets and after it said it deleted ~16k tweets before crashing, I still had 81k tweets. So I think it’s basically trying to delete the same tweets every time.

**Comment by n3wjack** [comment]
Every delete is an actual delete normally. In my case, I could see tweets disappear from my personal timeline as it was running.
You can track what it's doing in the network tab. The delete requests are visible there. If the header values are correct, they should succeed (response should be a 2xx status code).

Also set the unretweet value to true. This deletes retweets as well.
